packages/ocaml-systemd/ocaml-systemd.1.3/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+maintainer: "Juergen Hoetzel <juergen@archlinux.org>"
+authors: "Juergen Hoetzel <juergen@archlinux.org>"
+homepage: "https://github.com/juergenhoetzel/ocaml-systemd/"
+bug-reports:  "https://github.com/juergenhoetzel/ocaml-systemd/issues"
+dev-repo: "git+https://github.com/juergenhoetzel/ocaml-systemd.git"
+license: "LGPL-3.0-only WITH OCaml-LGPL-linking-exception"
+
+build: [
+  "dune"
+  "build"
+  "-j"
+  jobs
+  "@install"
+]
+depends: [
+  "ocaml"
+  "dune" {build & >= "2.0"}
```

Comment:
```suggestion
  "dune" {>= "2.0"}
```
Dune cannot be tagged as a build dependency due to a bug in dune: https://github.com/ocaml/dune/issues/2147